### DISCUSSION TITLE
请问怎么可以更改评论模板的样式呢？

### DISCUSSION THREAD

**Discussion by leewint** [walinejs/waline#1612] [Q&A]
请问怎么可以更改评论模板的样式呢？

**Comment by lizheming** [comment]
什么是评论模板？

**Reply by leewint** [reply]
![image](https://user-images.githubusercontent.com/44335558/203928888-e2bad456-1c72-4a9a-873e-d2dd8c345366.png)
就是想更改这个模板的样式，可以吗？

**Reply by lizheming** [reply]
https://waline.js.org/guide/client/style.html 不满足的自己写 CSS

**Reply by leewint** [reply]
好的，谢谢

**Comment by Mister-Hope** [comment]
如果你说的是邮件的那个，自己加CSS就行。
